src/app/ui/toc/symbology-stack.directive.js

```diff
@@ -219,6 +242,16 @@ function rvSymbologyStack($rootScope, $q, Geo, animationService, layerRegistry,
                 setTableDefinition(fullDef, defClause);
             }
 
+            if (defClause === undefined) {
+                if (self.block.validOIDs === undefined) {
+                    const proxyLoaded = $rootScope.$watch(() => self.block.proxyWrapper.state, (state, oldState) => {
```

Comment:
You should unwatch this when `(state === 'rv-loaded')` for performance reasons.

Reply:
donethanks